packages/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uma/common",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
If we're going to bump the version, should we manually write release notes? Or we could separate these into 2 PRs and do a full release on the following PR to generate release notes automatically.

Either way is fine with me -- we should just denote the version bump and reasoning in the release notes.

Reply:
I'm not sure opinionated on this tbh. I'm happy either way :)